src/main/java/se/sundsvall/oepintegrator/api/CaseResource.java

```diff
@@ -12,59 +13,77 @@
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import io.swagger.v3.oas.annotations.tags.Tag;
 import jakarta.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.List;
 import org.springframework.http.ResponseEntity;
 import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 import org.zalando.problem.Problem;
 import org.zalando.problem.violations.ConstraintViolationProblem;
 import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
+import se.sundsvall.oepintegrator.api.model.cases.CaseEnvelope;
 import se.sundsvall.oepintegrator.api.model.cases.SetStatusRequest;
 import se.sundsvall.oepintegrator.api.validation.ValidSetStatusRequest;
 import se.sundsvall.oepintegrator.utility.enums.InstanceType;
 
 @RestController
 @Validated
 @RequestMapping("/{municipalityId}/{instanceType}/cases")
-@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 @ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = {
 	Problem.class, ConstraintViolationProblem.class
 })))
+@ApiResponse(responseCode = "404", description = "Not found", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
```

Comment:
I don't think all endpoints will have the option to return 404 or am I missing something?

Reply:
What should we return if the id (externalId, flowInstanceId, whatever) doesn't result in a match?

So far all endpoints in this resource class are using some kind of identifier. What status code do you suggest if these identifiers doesn't return anything? 204?

Reply:
My concerns is limited to this `ResponseEntity<List<CaseEnvelope>> getCasesByFamilyId(..)`. I feel like this resource should return `HTTP 200 OK []`. 

But I guess there is an argument to be made that the familyId **can** result in no match and then 404 would be applicable.

Reply:
I don't care as long as we are doing the same everywhere.
200 for non-matches then?

(The OEP endpoint are returning a 404 if the family ID doesn't exist)